lib/noticed/delivery_methods/ios.rb

```diff
@@ -152,6 +152,15 @@ def development?
         end
       end
 
+      def valid_cert_path?
+        case cert_path
+        when StringIO
```

Comment:
Let's also add `File` here too, just in case the cert has already been read for whatever reason.

```suggestion
        when File, StringIO
```

Reply:
Great idea!